packages/blade/src/components/Typography/Heading/Heading.stories.tsx

```diff
@@ -81,6 +82,29 @@ WithColor.args = {
   color: 'feedback.notice.action.text.primary.default.lowContrast',
 };
 
+const Sup = isReactNative() ? HeadingComponent : 'sup';
+const WithMixedColorsTemplate: ComponentStory<typeof HeadingComponent> = () => {
+  return (
+    <Box>
+      <HeadingComponent>
+        Supercharge your business with the all‑powerful{' '}
+        <HeadingComponent as="span" color="badge.text.blue.lowContrast">
```

Comment:
lets not use things which have accidentally published 🙊

Reply:
:p. updated.